README.md

```diff
@@ -351,24 +348,23 @@ class MyCharm(CharmBase):
 
 
 def test_relation_data():
-    state_in = State(relations=[
-        Relation(
+    state_in = scenario.State(relations=[
+        scenario.Relation(
             endpoint="foo",
             interface="bar",
             remote_app_name="remote",
             local_unit_data={"abc": "foo"},
             remote_app_data={"cde": "baz!"},
         ),
     ])
-    ctx = Context(MyCharm,
-                  meta={"name": "foo"})
+    ctx = scenario.Context(MyCharm, meta={"name": "foo"})
 
-    state_out = ctx.run('start', state_in)
+    state_out = ctx.run(MyCharm.on.start, state_in)
 
     assert state_out.relations[0].local_unit_data == {"abc": "baz!"}
-    # you can do this to check that there are no other differences:
+    # You can do this to check that there are no other differences:
```

Comment:
I think this is one where we'll want to figure out how to do `relations["foo"]` instead of a numeral. Or should that be `relations["bar"]`?